pkg/controller.v1/tensorflow/pod.go

```diff
@@ -17,233 +17,49 @@ package tensorflow
 
 import (
 	"fmt"
-	"strconv"
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
-	common "github.com/kubeflow/common/pkg/apis/common/v1"
+	commonv1 "github.com/kubeflow/common/pkg/apis/common/v1"
+	tflogger "github.com/kubeflow/common/pkg/util"
+	train_util "github.com/kubeflow/common/pkg/util/train"
 	tfv1 "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1"
-	"github.com/kubeflow/tf-operator/pkg/common/jobcontroller"
-	tflogger "github.com/kubeflow/tf-operator/pkg/logger"
-	train_util "github.com/kubeflow/tf-operator/pkg/util/train"
 )
 
 const (
 	// tfConfig is the environment variable name of TensorFlow cluster spec.
 	tfConfig = "TF_CONFIG"
+	// gang scheduler name.
 
-	gangSchedulingPodGroupAnnotation = "scheduling.k8s.io/group-name"
+	//gangSchedulerName = "volcano"
+	//gangSchedulingPodGroupAnnotation = "scheduling.k8s.io/group-name"
 
 	// podTemplateRestartPolicyReason is the warning reason when the restart
 	// policy is set in pod template.
-	podTemplateRestartPolicyReason = "SettedPodTemplateRestartPolicy"
+	//podTemplateRestartPolicyReason = "SettedPodTemplateRestartPolicy"
 	// exitedWithCodeReason is the normal reason when the pod is exited because of the exit code.
-	exitedWithCodeReason = "ExitedWithCode"
+	//exitedWithCodeReason = "ExitedWithCode"
```

Comment:
It's safe to delete them. I think kubeflow/common reconcilePods() expose some these details. If not, please add to common.